### PR TITLE
added basic models

### PR DESCRIPTION
**Pull request by eric-ls**
made models for locations, requests, and recurrences

### CODE REVIEW

File: app/models/location.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#
+#  id         :integer          not null, primary key
+#  number     :string
+#  street     :string
+#  city       :string
+#  country    :string
+#  addr_name  :string
+#  apt_number :string
+#  state      :string
+#  zip        :string
```
Comment (warrenshen) [inline app/models/location.rb]: If you are going to be doing searches on this (on addresses in general) there's a good gem `geocoder` iirc.
https://github.com/alexreisner/geocoder

File: app/models/recurrence.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: recurrences
+#
+#  id         :integer          not null, primary key
+#  day        :integer
+#  starttime  :datetime
```
Comment (warrenshen) [inline app/models/recurrence.rb]: It's convention to do `start_time` and `end_time`.

File: app/models/recurrence.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#  endtime    :datetime
+#  frequency  :integer
+#  has_sent   :boolean
+#  startdate  :datetime
```
Comment (warrenshen) [inline app/models/recurrence.rb]: `start_date` as well (this is nit I guess).

File: app/models/location.rb (hunk 1 above)
Comment (HacksForHugs) [inline app/models/location.rb]: we only need to plug them into Onfleet (replate's operational software) as a json hash so we saved the fields as needed to do that

**Pull request merged**